Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -81,6 +81,13 @@ def elf_files
   end
 
   def self.relocation_formulae
-    ["patchelf"]
+    return @relocation_formulae if @relocation_formulae
+    f = ["patchelf"]
```

Comment:
Can you name this `formulae`? Wasn't obvious on first reading that you would be adding to this.